src/omv/ports/stm32/stm32fxxx_hal_msp.c

```diff
@@ -220,6 +220,13 @@ void HAL_MspInit(void)
     #endif
 
     #endif // DCMI_RESET_PIN || DCMI_PWDN_PIN || DCMI_FSYNC_PIN
+
```

Comment:
@kwagyeman Can you please add a 1/2 lines comment here on what it's doing and why ? Also, please make sure this doesn't affect the DAC/ADC pin.

Reply:
It doesn't affect the DAC/ADC pin. Tested that.

Reply:
Documented